src/txn/txn_rollback_to_stable.c

```diff
@@ -980,6 +980,109 @@ __rollback_to_stable_btree(WT_SESSION_IMPL *session, wt_timestamp_t rollback_tim
     return (ret);
 }
 
+/*
+ * __rollback_evict_exclusive_apply --
+ *     Apply an eviction helper to all open dhandles.
+ */
+static int
+__rollback_evict_exclusive_apply(WT_SESSION_IMPL *session, bool *files_acquired,
```

Comment:
Could you use {{\_\_wt_conn_btree_apply)) instead of this function?

Reply:
I don't think so. We need `files_acquired`.